chrome/content/zotero/xpcom/translation/translate.js

```diff
@@ -1528,7 +1535,8 @@ Zotero.Translate.Base.prototype = {
 		this._aborted = false;
 		this.saveQueue = [];
 		
-		Zotero.debug("Translate: Parsing code for "+translator.label, 4);
+		Zotero.debug("Translate: Parsing code for " + translator.label
+			+ " -- " + translator.translatorID + " -- " + translator.lastUpdated, 4);
```

Comment:
I tried that, but it looks really dense in the debug output. Personally, I find this style more pleasant. Try it out if you have time.